### PR TITLE
POC of d3 fhir mapping graph

### PR DESCRIPTION
**Pull request by simonvadee**

### CODE REVIEW

File: d3-fhir-graph/src/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
```
Comment (Jasopaum) [inline d3-fhir-graph/src/index.js:5]: I think we can remove this

File: d3-fhir-graph/src/App.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```
Comment (Jasopaum) [inline d3-fhir-graph/src/App.test.js:1]: We don't need test files for React apps here

File: d3-fhir-graph/src/App.js [hunk 3]
```diff
@@ -0,0 +1,624 @@
...
+    ];
+  }, []);
+  console.log(nodes, links);
+  return { nodes, links };
+};
+
+// parse(mapping);
+const data = parse(mapping);
+
+// const data = {
```
Comment (Jasopaum) [inline d3-fhir-graph/src/App.js:65]: Can we remove the mock data?

File: d3-fhir-graph/src/index.js (hunk 1 above)
Comment (simonvadee) [reply d3-fhir-graph/src/index.js:5]: done

**Pull request merged**